src/backend/src/utils/validation.utils.ts

```diff
@@ -11,23 +12,32 @@ export const nonEmptyString = (validationObject: ValidationChain): ValidationCha
 };
 
 export const isRole = (validationObject: ValidationChain): ValidationChain => {
-  return validationObject.isString().isIn(['APP_ADMIN', 'ADMIN', 'HEAD', 'LEADERSHIP', 'MEMBER', 'GUEST']);
+  return validationObject
+    .isString()
+    .isIn([RoleEnum.ADMIN, RoleEnum.APP_ADMIN, RoleEnum.GUEST, RoleEnum.HEAD, RoleEnum.LEADERSHIP, RoleEnum.MEMBER]);
```

Comment:
can you rearrange this so that its like what it was before, i.e. app admin admin head leadership member guest, that way its easy to tell what the rankings symbolize